dist/debian/control.template

```diff
@@ -47,10 +47,20 @@ Conflicts: prometheus-node-exporter
 Description: Prometheus exporter for machine metrics
  Prometheus exporter for machine metrics, written in Go with pluggable metric collectors.
 
+Package: %{product}-jmx
+Depends: ${misc:Depends}
+Architecture: all
+Priority: optional
+Section: oldlibs
+Description: transitional package
+ This is a transitional package. It can safely be removed.
+
 Package: %{product}
 Section: metapackages
 Architecture: any
 Depends: %{product}-server (= ${binary:Version})
+ , %{product}-tools (= ${binary:Version})
+ , %{product}-tools-core (= ${binary:Version})
```

Comment:
why are we adding `scylla-tools` and `scylla-tools-core` back as dependencies of `scylla` ? they were previously removed in b8634fb244ab08c2ffa66700745b06e21ee57f30 .

Reply:
This was mistakenly happened because of the patch based on older code, fixed.